src/installer/tests/Assets/TestUtils/TestProjects.props

```diff
@@ -13,5 +13,7 @@
       win-x86 tests that assumed a win-x64 app host RID based on the runner SDK.
     -->
     <AppHostRuntimeIdentifier>$(TestTargetRid)</AppHostRuntimeIdentifier>
+
+    <NewtonsoftJsonVersion>13.0.1</NewtonsoftJsonVersion>
```

Comment:
Is this necessary? I would have thought that this gets propagated from `Versions.props`...

Reply:
This is used by host test projects, which are published during the test runs. At that point artifices/tests is copied to remote machine (helix) with TestUtils, with no access to `Versions.props`. https://github.com/dotnet/runtime/blob/3a1fd4d2e5cbb4850ae608a5867c79c60781da25/src/installer/tests/Assets/TestUtils/TestProjects.props#L2-L5

Reply:
Thanks a lot for the explanation!